### ISSUE TITLE
Add function to not display hidden files/folders

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#92]
```

For the folder plug-in, it would be nice to have a function to skip the hidden 
files/folders (starting with a ".") when including subfolders or not.


```

Original issue reported on code.google.com by `onyx3...@gmail.com` on 9 May 2011 at 6:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `yendo0206` on 9 May 2011 at 2:51
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision 4103370e9d3a.
```

Original comment by `yendo0206` on 11 May 2011 at 2:15
- Changed state: **Fixed**
